src/language-html/embed/utils.js

```diff
@@ -23,9 +23,16 @@ function shouldHugJsExpression(ast, options) {
     rootNode &&
     (rootNode.type === "ObjectExpression" ||
       rootNode.type === "ArrayExpression" ||
+      // Vue
       ((options.parser === "__vue_expression" ||
         options.parser === "__vue_ts_expression") &&
         (rootNode.type === "TemplateLiteral" ||
+          rootNode.type === "StringLiteral")) ||
+      // Angular
+      ((options.parser === "__ng_binding" ||
+        options.parser === "__ng_directive") &&
+        (rootNode.type === "TemplateLiteral" ||
+          rootNode.type === "TaggedTemplateExpression" ||
           rootNode.type === "StringLiteral")))
```

Comment:
```suggestion
      // Vue
      ((options.parser === "__vue_expression" ||
        options.parser === "__vue_ts_expression" ||
        // Angular
        options.parser === "__ng_binding" ||
        options.parser === "__ng_directive") &&
        (rootNode.type === "TemplateLiteral" ||
          rootNode.type === "StringLiteral")))
```

Reply:
Sure; I will remove that node type (may I ask why, out of curiosity?); and I might remove the `// Vue` and `// Angular` comments I previously added as well because the Angular & Vue checks are siblings.

Reply:
TaggedTemplateExpression can be complicated in it's tag. But we can reconsider if requested, if we do want add it, should apply to vue too.